src/app/layout/footer/network-link.component.ts

```diff
@@ -0,0 +1,5 @@
+export class NetworkLink {
+    constructor(
+      public name: string,
+      public link: string) { }
+}
```

Comment:
The name of this class could be: SocialLink
Please, place this in app/shared/models
The name of the file would be: social-link.model.ts
Use 2 spaces for indentation. If you install Prettier, you can format the code using the command: alt + shift + f